lib/iterable_flutter.dart

```diff
@@ -27,8 +27,14 @@ class IterableFlutter {
     _channel.setMethodCallHandler(nativeMethodCallHandler);
   }
 
-  static Future<void> setEmail(String email) async {
-    await _channel.invokeMethod('setEmail', email);
+  static Future<void> setEmail(String email, String jwt) async {
```

Comment:
I would make jwt nullable in order to communicate that it isn't strictly required. Some implementations either don't care about having a JWT, or just want to get something quick put together to ensure the package works.